mdetsims/sim_utils.py

```diff
@@ -509,11 +517,15 @@ def _get_dxdy(self, others=None, min_dist=0):
                 xind * dg + dg/2 - self.pos_width)
         else:
             while True:
-                dx, dy = self.rng.uniform(
-                    low=-self.pos_width,
-                    high=self.pos_width,
-                    size=2)
-
+                if self.pair_sim is not None:
```

Comment:
We shouldhave pair_sim be a pure boolean value. So either it is True or False. The default should be False.